proxies/nodejs/handlers/client.ts

```diff
@@ -8,23 +8,23 @@ type ClientRequestBody = {
 }
 
 export const handleClient = async (ctx: Koa.ParameterizedContext, clients: { [key: string]: DVCClient }) => {
-  const body = <ClientRequestBody>ctx.request.body
-  if (body.clientId === undefined) {
-    ctx.status = 400
-    ctx.body = {
-      error: "Invalid request: missing clientId"
+    const body = <ClientRequestBody>ctx.request.body
+    if (body.clientId === undefined) {
+        ctx.status = 400
+        ctx.body = {
+            error: 'Invalid request: missing clientId'
+        }
+    } else {
```

Comment:
noticed you resolved the convo, did you decide against refactoring this out of the else clause?

Reply:
oops, I only looked at `location.ts` and missed this